Vagrantfile

```
@@ -36,15 +36,17 @@ Vagrant.configure(2) do |config|
     config.ssh.forward_agent = true
     config.vm.provision "shell", path: "scripts/provision.sh", privileged: false
     config.vm.provider "virtualbox" do |vb, override|
-       override.vm.box = "ubuntu/trusty64"
+       override.vm.box = "ubuntu/bionic64"
        # find out on which host os we are running
        host = RbConfig::CONFIG['host_os']
        vb.customize ["modifyvm", :id, "--ioapic", "on"]
-       vb.memory = get_memory_setting(host)
+       vb.memory = "4096"
```

I think the old value for memory is better because not all computers have > 4 GB of memory

```suggestion
       vb.memory = get_memory_setting(host)
```